.devcontainer/docker-compose-local.yaml

```diff
@@ -34,7 +34,7 @@ services:
       - localstack-data:/var/lib/localstack
 
   ray:
-    image: rayproject/ray:2.30.0-py311-cpu-aarch64
+    image: rayproject/ray:2.30.0-py311-cpu${ARCH}
```

Comment:
Maybe `-${ARCH}` and remove the dash later on?

Reply:
As the non-aarch64 image does not have any suffix, this would break things. As I am not 100% sure this is a standard (e.g. are suffixes starting with `_` used too?) perhaps adding the dash explicitly in the suffix will work better. WDYT?